### PR TITLE
Fixed core and documentation compliance

### PR DESCRIPTION
**Pull request by Tom-Kingstone**
### Issues addressed by this PR

Closes #

moved `BoundingRectangle.cs` to a new `CurveLoop` folder, and fixed some docs attributes

### CODE REVIEW

**Comment by Tom-Kingstone** [comment]
@BHoMBot check core
@BHoMBot check compliance

**Comment by Tom-Kingstone** [comment]
@BHoMBot check required

File: Revit_Core_Engine/Revit_Core_Engine.csproj [hunk 1]
```diff
@@ -40,8 +40,7 @@
 	<DefineConstants>REVIT2024</DefineConstants>
   </PropertyGroup>
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="	xcopy &quot;$(TargetDir)$(TargetFileName)&quot; &quot;C:\ProgramData\BHoM\Assemblies&quot; /Y
-					xcopy &quot;$(ProjectDir)Files\Families&quot; &quot;C:\ProgramData\BHoM\Resources\Revit\Families&quot; /Y /I /E" />
+    <Exec Command="	xcopy &quot;$(TargetDir)$(TargetFileName)&quot; &quot;C:\ProgramData\BHoM\Assemblies&quot; /Y&#xD;&#xA;					xcopy &quot;$(ProjectDir)Files\Families&quot; &quot;C:\ProgramData\BHoM\Resources\Revit\Families&quot; /Y /I /E" />
```
Comment (FraserGreenroyd) [inline Revit_Core_Engine/Revit_Core_Engine.csproj:43]:
```suggestion
    <Exec Command="	xcopy &quot;$(TargetDir)$(TargetFileName)&quot; &quot;$(ProgramData)\BHoM\Assemblies&quot; /Y&#xD;&#xA;					xcopy &quot;$(ProjectDir)Files\Families&quot; &quot;$(ProgramData)\BHoM\Resources\Revit\Families&quot; /Y /I /E" />
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
Check my suggestion works but we should use the `$(ProgramData)` rather than reference the `C:`

**Comment by Tom-Kingstone** [comment]
@BHoMBot check core
@BHoMBot check compliance

File: Revit_Core_Engine/Objects/ActiveViewVisibilityContext.cs [hunk 2]
```diff
@@ -37,7 +37,7 @@ public class ActiveViewVisibilityContext : IExportContext
         [Description("hostDocument refers to the document that owns the active view. targetDocument can take three values:" +
                      "\n- same as hostDocument - visible elements of the host document are then collected" +
                      "\n- document linked in the host document - elements of that linked document visible in the active view of the host document are then collected" +
-                     "\n- null - all elements of all documents, host and links, are then collected")]
+                     "\n- null - all elements of all documents, host and links, are then collected.")]
```
Comment (pawelbaran) [inline Revit_Core_Engine/Objects/ActiveViewVisibilityContext.cs:40]: Are we sure the dot makes any sense here? 😉

**Review by FraserGreenroyd — APPROVED** [review]
This looks roughly ok now, not withstanding @pawelbaran comment which is valid, but I'm going for a "what gets us more compliant" based on the changes to the other files as well. So not perfect, but a step in the right direction so we'll take it, thanks @Tom-Kingstone

**Comment by FraserGreenroyd** [comment]
@BHoMBot check null-handling
@BHoMBot check serialisation

**Pull request merged**